pyblish_qml/control.py

```diff
@@ -1052,14 +1052,12 @@ def iterator(plugins, context):
                 yield plugin, instance
 
         else:
-        
-     
             # When filtering to families at least a single instance with
             # that family must be active in the current publish
-            if "*" not in plugin.families:
-                if not any(instance.data.get("publish") is not False 
-                           for instance in instances):
-                    continue
-                    
-        
+            no_wildcard = "*" not in plugin.families
+            no_active_instance = not any(inst.data.get("publish") for inst in instances)
```

Comment:
@mottosso note that this is now a different check than what `logic.py` does in `pyblish-base` because there it explicitly checks for `is not False`. This could potentially start behaving different.

Reply:
Yes. Do try this locally first, then we'll need to update Lite and Base to follow suit. It's a backwards incompatible change, so odds are we'll put it behind a `PYBLISH_SOMETHING_SOMETHING` environment flag to be explicitly enabled.